src/django_components/slots.py

```diff
@@ -56,9 +56,6 @@ class SlotFunc(Protocol, Generic[TSlotData]):
     def __call__(self, ctx: Context, slot_data: TSlotData, slot_ref: "SlotRef") -> SlotResult: ...  # noqa E704
 
 
-SlotContent = Union[SlotResult, SlotFunc[TSlotData], "Slot[TSlotData]"]
```

Comment:
Needed to move this to avoid using the forward reference (AKA type inside a string like `"Slot[TSlotData]"`), as that was causing some issues with the weakrefs. Don't remember 100% what it was.